variables.tf

```
@@ -42,7 +42,7 @@ variable "policy" {
     Statement = list(object({
       Sid       = string
       Effect    = string
-      Principal = map(any)
+      Principal = any
```

As mentioned in [my comment](https://github.com/scaleway-terraform-modules/terraform-scaleway-bucket/pull/60#issuecomment-2741221228), there's no need to change the type of this attribute: It is possible to use a wild-card as principal using the following:
```
...
          Principal = {
            SCW = "*"
          },
...
```

So this change is not needed.

```suggestion
      Principal = map(any)
```